package.json

```diff
@@ -256,6 +266,11 @@
         "command": "one.editor.cfg.setDefaultValues",
         "title": "ONE: Set Default Values",
         "category": "ONE"
+      },
+      {
+        "command": "one.editor.mpq.showFromDefaultExplorer",
+        "title": "Create MPQ json",
+        "category": "ONE"
```

Comment:
I think this command may be shown in the menu of ONE Explorer, rather than the default explorer.
You can refer to the code below.

https://github.com/Samsung/ONE-vscode/blob/a00161070e2a4de17badf5ea9113f9131156d058/package.json#L250-L254

Reply:
ok. I'll try.

Reply:
@dayo09 
I've call of this option from ONE Explorer, but there are some issues:
the command should be available only for `circle` files, but in ONE Explorer and `view/item/context` only `viewItem` is available which doesn't have `circle` entry. Am i missing something? or may be there is another convenient way to recognize `circle` model?
As of now the PR returns immediately from creation function as soon as it recognizes that the file is not `circle` model.

Reply:
https://github.com/Samsung/ONE-vscode/blob/a00161070e2a4de17badf5ea9113f9131156d058/package.json#L362-L366

You can refer the code above.

You can filter it out by "viewItem == product && resourceExtname == .circle"

Reply:
@dayo09 
Unfortunately,
seems like `resourceExtname` ignores active item  in `OneExplorer` and returns extension of active item only for default `Explorer` , so i failed with the code above. Please see https://github.com/Samsung/ONE-vscode/blob/460f46750dfcd164510a21acde60ebb745bb15d5/src/OneExplorer/OneExplorer.ts#L464.
AFAIK the only option available is to use `viewItem`, but it needs some effort to distinguish `circle` from other products.
But i may be wrong.

Reply:
Ok. Thank you.